File: 프로그래머스/unrated/155652. 둘만의 암호/둘만의 암호.js
function solution(s, skip, indexs) {
    const alphabet = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z']
    let answer = "";
    
    // 제거부터 
    for (let str of skip) {
        const index = alphabet.indexOf(str);
        
        alphabet.splice(index, 1);
    }
     
    // 뒤로 밀기
    for (let str of s) {
        const index = alphabet.indexOf(str);
        
        answer += alphabet[(index + indexs) % alphabet.length];
    }
    
    return answer;
}

File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.py
def solution(s, skip, index): 
    alphabet = ['a', 'b', 'c', 'd', 'e', 'f', 'g','h', 'i', 'j', 'k', 'l', 'm','n','o','p','q','r','s','t','u','v','w','x','y','z']
    
    for i in skip:
        skipIndex = (alphabet.index(i))
        alphabet.pop(skipIndex)
        
    result = []
 
    for j in s:
        newIndex = (alphabet.index(j) + index) % len(alphabet)
        result.append(alphabet[newIndex])
        
    
    return (''.join(result))